prompts/litellm/azure-gpt4o-analysis.yml

```
@@ -0,0 +1,48 @@
+model_settings:
+  temperature: 0.2
+  max_tokens: 1500
+  top_p: 0.9
+prompt: |
+  Analyze the following document and provide comprehensive insights:
+
+  Document Type: {doc_type}
+  Content: {content}
+
+  Please provide:
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)